docs/REQFILE.md

```diff
@@ -0,0 +1,115 @@
+# Requirement Files Guide
+
+## General
+1. **Requirement files** hold the details of the team you're trying to optimize.
+2. You can optimize up to **10 cookies** in each file.
+3. Use `!reqdownload` to download a default file and view an example.
+
+## Requirements
+4. Requirements are specified under each cookie in a **yaml file**.
+5. Each cookie can accept a collection of **validity requirements** and one **objective requirement**.
+
+## Bot Moderator Permissions and Capabilities
+
+### Actions on Behalf of Others
+1. **Uploading Requirements for Users**: Moderators can upload requirement files directly into other users' inventories.
+2. **Optimizing Using User's Topping Inventory**: Moderators have the capability to optimize configurations using the topping inventory of other users.
+
+### Copying Requirement Files
+1. **Copying Files**: Moderators can copy requirement files between users, ensuring efficient management and sharing of configurations within the guild.
```

Comment:
copying can be done by both mods & non-mods, but it looks different:
Non-mod, copy a default req into a private copy
Mod: copy one of your private reqs into someone elses private reqs